detox/package.json

```diff
@@ -73,14 +73,14 @@
     "bunyan-debug-stream": "^3.1.0",
     "caf": "^15.0.1",
     "chalk": "^4.0.0",
-    "child-process-promise": "^2.2.0",
-    "detox-copilot": "^0.0.27",
+    "detox-copilot": "^0.0.24",
```

Comment:
Failing tests might be related to this downgrade...

Reply:
Could you try to update it @matinzd ?